packages/babel-generator/src/printer.js

```diff
@@ -513,7 +513,7 @@ export default class Printer {
     );
   }
 
-  _printComment(comment) {
+  _printComment(comment, node) {
```

Comment:
Ok agreed, I actually think of that, but I was done with this implementation by the time you commented 😅 

I'll try the approach you suggested!